src/logic/embedding.lean

```diff
@@ -217,6 +217,10 @@ lemma refl_to_embedding {α : Type*} : (equiv.refl α).to_embedding = function.e
 lemma trans_to_embedding {α β γ : Type*} (e : α ≃ β) (f : β ≃ γ) :
   (e.trans f).to_embedding = e.to_embedding.trans f.to_embedding := rfl
 
+lemma swap_embedding_apply {α β : Type*} [decidable_eq α] [decidable_eq β] (f : α ↪ β) (x y z : α) :
```

Comment:
This lemma should be `function.embedding.swap_apply` so that we can use dot-notation.